src/Compilers/Core/Portable/Operations/IMethodBodyOperation.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents a method body operation.
+    /// <para>
+    /// Current usage:
+    ///  (1) C# method body for non-constructor
```

Comment:
Are we going to eventually unify the IOperation APIs added in this PR to even include VB methods?

Reply:
We can discuss this at the design meeting this week. We didn't make this decision at the last meeting. Making this change will be a breaking change because we already return IOperation nodes for method declarations in VB, we return IBlockOperation and in VB there is no such thing as expression body. 

---
In reply to: [170774730](https://github.com/dotnet/roslyn/pull/25031#discussion_r170774730) [](ancestors = 170774730)